sentences/fi/_common.yaml

```diff
@@ -40,14 +22,41 @@ lists:
         out: "blue"
       - in: "vihreä"
         out: "green"
+      - in: "violetti"
+        out: "purple"
+      - in: "ruskea"
+        out: "brown"
+  brightness:
+    range:
+      type: "percentage"
+      from: 0
+      to: 100
+  temperature:
+    range:
+      type: "temperature"
+      from: 0
+      to: 100
+  temperature_unit:
+    values:
+      - in: "(c|celsius)"
+        out: "celsius"
+      - in: "(f|fahrenheit)"
+        out: "fahrenheit"
 expansion_rules:
-  laita: "(kytke | käynnistä | aseta)"
+  käynnistä: "(kytke | käynnistä | aseta)"
   mikä: "(mikä | mikä on)"
   temp: "(lämpö | lämpötila)"
-  area: "{area}"
-  brightness: "{brightness} [Prosenttia]"
-  name: "{name}"
-  turn: "(käännä | kytke)"
-  turn_on: "(laita | laita päälle)"
-  turn_off: "(poista | laita pois)"
-skip_words: []
+  area: "(alue)"
+  kirkkaus: "(kirkkaus)"
+  nimi: "({name})"
+  avaa: "(avaa)"
+  sulje: "(sulje)"
+  käännä: "(käännä | kytke)"
```

Comment:
This is a genuine question, do you use "käännä lamppu päälle"? If that feels natural, lets leave it, but if it feels a bit "translated", lets take it off, right? 😄